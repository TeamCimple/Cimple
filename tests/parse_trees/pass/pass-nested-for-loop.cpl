int main() {
	int i;

	for(i = 1; i < 100; i++) {
		for (j = 1; j < 100; j++) {
			return 0;
		}
	}
}
