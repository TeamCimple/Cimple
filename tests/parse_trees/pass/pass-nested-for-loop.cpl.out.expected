FuncDecl(Name: Var(main) ReturnType: int Parameters:  Body: CompoundStatement(Declarations: (int Var(i),),  StatementList: FOR EQUALS(Identifier(i), Int(1)) Compare(Identifier(i),LESS_THAN,Int(100)) Postfix(Identifier(i),++) CompoundStatement(Declarations:  StatementList: FOR EQUALS(Identifier(j), Int(1)) Compare(Identifier(j),LESS_THAN,Int(100)) Postfix(Identifier(j),++) CompoundStatement(Declarations:  StatementList: RETURN(Int(0)))))) 
